symfony/twig-bundle/5.4/config/packages/twig.yaml

```diff
@@ -0,0 +1,4 @@
+twig:
+    default_path: '%kernel.project_dir%/templates'
+    debug: '%kernel.debug%'
+    strict_variables: '%kernel.debug%'
```

Comment:
It looks like this file is a bit simpler in the current, 5.0 recipe: https://github.com/symfony/recipes/blob/master/symfony/twig-bundle/5.0/config/packages/twig.yaml

Everything else looks good

Reply:
updated, thanks